examples/limitrange/README.md

```diff
@@ -1,4 +1,147 @@
-Please refer to this [doc](https://github.com/GoogleCloudPlatform/kubernetes/blob/620af168920b773ade28e27211ad684903a1db21/docs/design/admission_control_limit_range.md#kubectl).
+Limit Range
+========================================
+By default, pods run with unbounded CPU and memory limits.  This means that any pod in the
+system will be able to consume as much CPU and memory on the node that executes the pod.
+
+Users may want to impose restrictions on the amount of resource a single pod in the system may consume
+for a variety of reasons.
+
+For example:
+
+1. Each node in the cluster has 2GB of memory.  The cluster operator does not want to accept pods
+that require more than 2GB of memory since no node in the cluster can support the requirement.  To prevent a
+pod from being permanently unscheduled to a node, the operator instead chooses to reject pods that exceed 2GB
+of memory as part of admission control.
+2. A cluster is shared by two communities in an organization that runs production and development workloads
+respectively.  Production workloads may consume up to 8GB of memory, but development workloads may consume up
+to 512MB of memory.  The cluster operator creates a separate namespace for each workload, and applies limits to
+each namespace.
+3. Users may create a pod which consumes resources just below the capacity of a machine.  The left over space
+may be too small to be useful, but big enough for the waste to be costly over the entire cluster.  As a result,
+the cluster operator may want to set limits that a pod must consume at least 20% of the memory and cpu of their
+average node size in order to provide for more uniform scheduling and to limit waste.
+
+This example assumes that you have a functional Kubernetes setup.
+
+This example demonstrates how limits can be applied to a Kubernetes namespace to control
+min/max resource limits per pod.  In addition, this example demonstrates how you can
+apply default resource limits to pods in the absence of an end-user specified value.
+
+For a detailed description of the Kubernetes resource model, see [Resources](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/resources.md)
+
+Step 1: Create a namespace
+-----------------------------------------
+This example will work in a custom namespace to demonstrate the concepts involved.
+
+Let's create a new namespace called limit-example:
+
+```shell
+$ cluster/kubectl.sh create -f namespace.yaml
```

Comment:
This should be run from the kubernetes root, right?  So it should be `-f examples/limitrange/namespace.yaml`.